.mergify.yml

```diff
@@ -16,10 +20,4 @@ pull_request_rules:
     actions:
       comment:
         message: Thank you for contributing! Your pull request is now going on the merge train (choo choo! Do not click update from main anymore, and be sure to [allow changes to be pushed to your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)).
-      queue:
-        name: default_queue
```

Comment:
Oops! In hindsight, I think we shouldn't have removed this part:

```
queue:
  name: default_queue
```

That's probably the command to enqueue the PR for actual merging.

Noticing this after noticing that the Weblate PR didn't merge itself.

Reply:
Sorry! Made a PR: #5741 !